docs/index.rst

```diff
@@ -252,6 +252,41 @@ Support for a growing number of locales (see ``locales.py`` for supported langua
     >>> future.humanize(a, locale='ru')
     'через 2 час(а,ов)'
 
+Dehumanize
+~~~~~~~~
```

Comment:
The underline length needs to match the title (causes the lint action to fail).

Reply:
Linting issue fixed, as well as documentation updated.